### PR TITLE
add new term 'Ductus venosus agenesis'

### LINKED ISSUE #7363: Ductus venosus agenesis
**Preferred term label:**
Ductus venosus agenesis

**Synonyms**
Agenesis of the ductus venosus

**Definition (free text, please give PubMed ID)**
A developmental defect characterized by lack of formation of the ductus venosus (a shunt that allows oxygenated blood in the umbilical vein to bypass the liver in the fetal circulation). Agenesis of the ductus venosus can manifest in two different morphologic patterns: The umbilical vein drains exclusively into the left branch of the intrahepatic portal vein or the umbilical vein drains into the inferior vena cava or directly into the right atrium by-passing the liver completely.
PMID: 10765517
**Parent term (use hpo.jax.org/app)**
Abnormality of the fetal cardiovascular system HP:0010948

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7363

### CODE REVIEW

**Comment by pnrobinson** [comment]
I am going to close this since we were advised to modify in the recent workshop.
